samples/actions/serverless.yaml

```diff
@@ -0,0 +1,65 @@
+## Cloning the Repository
+- action: 
+  uses: git-clone
+  with:
+    branch: main
+    repo: Product-Management
+    connector: cd-demo
+
+---
+## Download Manifest
+- action: 
+  uses: harness-download
+
+---
+
+## Serverless Package
+- action:
+  uses: serverless-package
+  with:
+    command: "" ## Optional
+    image-pull-policy: always
+    user: 1000
+    memory: 10mb
+    cpu: 0.25
+    args:
```

Comment:
@thisrohangupta How do we want to take care of downloading of artifacts and feeding into this serverless.yaml ? 
Right now , its done inside serverless package step by us before executing serverless package command , but that's not a good design here. Each plugin should have well defined responsibilities and downloading of artifacts should not be the responsibility of serverless package plugin. Customers could always script the downloading of artifacts , and then scripting the part of putting that artifact inside serverless.yml. But the question is do we want customers to script it ? Should we have a seperate plugin for handling that ? I am not sure if this is urgent , but we should think about handling artifacts in our services via an independent plugin

Reply:
We use custom expressions inside serverless manifest to feed artifact like "<+artifact.image>" and "<+artifact.path>".  If we have an independent plugin for downloading artifacts, we could generalize some expressions for the path where the artifact downloads and use that expressions instead , which could be used irrespective of the swimlane

Reply:
@piyush-bhuwalka we will run arifact collection as an independent plugin.